pkg/descheduler/evictions/evictions.go

```diff
@@ -134,6 +132,14 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, node *v1.Node,
 		r := eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "sigs.k8s.io.descheduler"})
 		r.Event(pod, v1.EventTypeNormal, "Descheduled", fmt.Sprintf("pod evicted by sigs.k8s.io/descheduler%s", reason))
 		metrics.PodsEvicted.With(map[string]string{"result": "success", "strategy": strategy, "namespace": pod.Namespace}).Inc()
+		var ownerRefName, ownerRefKind string
+		ownerRefs := pod.OwnerReferences
+		for _, ownerRef := range ownerRefs {
+			ownerRefName = ownerRef.Name
+			ownerRefKind = ownerRef.Kind
+		}
+		metrics.DuplicatePods.With(map[string]string{"name": ownerRefName,
+			"kind": ownerRefKind, "namespace": pod.Namespace, "node": node.Name}).Dec()
```

Comment:
This will get invoked by every strategy calling `EvictPod`, not just by `RemoveDuplicatePods`. Was there an issue decreasing the metrics right after `EvictPod` in `RemoveDuplicatePods`? Also, I'd rather expect `RemoveDuplicatePods` to keep count of `duplicated` and `evicted` pods and then set the metrics to `duplicated` - `evicted`. Is there any value in watching eviction of duplicated pods by one? Pods get evicted quite quickly since a strategy does not wait until a pod is terminated. It might be interesting to see it in case the strategy evicts hundreds of pods. Although, that might indicate a serious problem in the cluster (e.g. strategy constrains do not balance the pods properly).